workloads/kube-burner/common.sh

```diff
@@ -166,17 +166,66 @@ snappy_backup() {
  store_on_elastic
 }
 
-remove_update_taint() {
-  # This is only here until BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035005 gets resolved
-  oc adm taint nodes UpdateInProgress:PreferNoSchedule- --all || true
+get_network_type() {
+if [[ $NETWORK_TYPE == "OVNKubernetes" ]]; then
+  network_ns=openshift-ovn-kubernetes
+else
+  network_ns=openshift-sdn
+fi
+echo $network_ns
 }
 
+check_metric_to_modify() {
+  export div_by=1
+  echo $config | grep -i memory
+  if [[ $? == 0 ]]; then 
+   export div_by=1048576
+  fi
+  echo $config | grep -i latency
+  if [[ $? == 0 ]]; then
+    export div_by=1000
+  fi
+}
+
+run_benchmark_comparison() {
+   if [[ -n ${ES_SERVER} ]] && [[ -n ${COMPARISON_CONFIG} ]]; then
+     log "Installing touchstone"
+     install_touchstone
+     NUM_LINES=${NUM_LINES:-10}
+     get_network_type
+     export TOUCHSTONE_NAMESPACE=${TOUCHSTONE_NAMESPACE:-"$network_ns"}
+     res_output_dir="/tmp/${WORKLOAD}-${UUID}"
```

Comment:
Are we sure we need this TOUCHSTONE_NAMESPACE variable?, keep in mind that the generated spreadsheet has column filters so finding/selecting a specific namespace shouldn't be a big deal.